github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml

```
@@ -576,3 +576,44 @@ periodics:
       - name: token
         secret:
           secretName: oauth-token
+
+- name: periodic-project-infra-mirror-istioctl
+  cron: "25 1 * * 1"
+  decorate: true
+  annotations:
+      testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  extra_refs:
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+    workdir: true
+  labels:
+    preset-shared-images: "true"
+  spec:
```

You need also to specify in which cluster it is going to run.
```suggestion
  cluster: prow-workloads
  spec:
```